lib/navigation-bar/index.tsx

```diff
@@ -14,20 +14,34 @@ import { viewExternalUrl } from '../utils/url-utils';
 import appState from '../flux/app-state';
 import DialogTypes from '../../shared/dialog-types';
 
-export class NavigationBar extends Component {
+import { toggleNavigation } from '../state/ui/actions';
+
+import * as S from '../state';
+import * as T from '../types';
+
+type StateProps = {
+  showNavigation: boolean;
+};
+
+type DispatchProps = {
+  onOutsideClick: () => any;
+  toggleNavigation: Function;
```

Comment:
this is fine, though I have preferred the `() => any` style both for consistency but also for one more thing: `Function` doesn't convey that this is a "fire-and-forget" action the same way that `() => any` does to me.

`Function` doesn't specify the argument types while `() => any` communicates "this function expects no arguments" and sometimes that's handy to know, especially if jumping into the code and trying to work with what's there.

since these are right next to each other, did you have a reason to keep them separate from each other?

Reply:
No, literally just copy and pasting things without understanding the difference. I was going to ask which is preferred and why!